src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -0,0 +1,185 @@
+  const open = Boolean(anchorEl);
+
+  const { data, loading, fetchMore } = useContactsQuery({
+    variables: {
+      accountListId: accountListId ?? '',
+      contactsFilters: {
+        ...sanitizedFilters,
+        appeal: [appealId ?? ''],
+        appealStatus,
+        wildcardSearch: searchTerm as string,
+      },
+    },
+    skip: !accountListId || !appealStatus,
+  });
+
+  const cardContentRef = useRef<HTMLDivElement>();
+
+  const [{ canDrop }, drop] = useDrop(() => ({
+    accept: 'contact',
+    collect: (monitor) => ({
+      canDrop: !!monitor.canDrop(),
+    }),
+  }));
+
+  const handleColumnHeaderClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleSelectAll = () => {
+    setAnchorEl(null);
+    // TODO implement select all
+  };
+
+  const handleDeselectAll = () => {
+    setAnchorEl(null);
+    // TODO implement deselect all
+  };
+
+  const totalContacts = data?.contacts.totalCount || 0;
+
+  return loading && !data ? (
+    <CircularProgress />
+  ) : (
+    <Card>
+      <ContainerBox p={2} data-testid="column-header" color={color}>
+        <Box width="80%">
+          <ColumnTitle variant="h6">{title}</ColumnTitle>
+        </Box>
+        <Box display="flex" alignItems="center">
+          <Typography>{data?.contacts.totalCount || 0}</Typography>
+
+          <IconButton
+            aria-label={t('Column select')}
+            id="demo-positioned-button"
+            aria-controls={open ? 'demo-positioned-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
+            onClick={handleColumnHeaderClick}
+          >
+            <MoreVertIcon />
+          </IconButton>
+
+          <Menu
+            id="demo-positioned-menu"
+            aria-labelledby="demo-positioned-button"
+            anchorEl={anchorEl}
+            open={open}
+            onClose={handleClose}
+            anchorOrigin={{
+              vertical: 'top',
+              horizontal: 'left',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'left',
+            }}
+          >
+            <MenuItem onClick={handleSelectAll}>
+              {totalContacts <= 1
+                ? t('Select {{count}} contact', { count: totalContacts })
+                : t('Select all {{count}} contacts', { count: totalContacts })}
+            </MenuItem>
+            <MenuItem onClick={handleDeselectAll}>{t('Deselect All')}</MenuItem>
+          </Menu>
+        </Box>
+      </ContainerBox>
+      <StyledCardContent
+        style={{
+          height: `calc(100vh - ${navBarHeight} - ${headerHeight} - ${appealHeaderInfoHeight} - 87px)`,
+        }}
+      >
+        <CardContentInner
+          canDrop={canDrop}
+          gridTemplateRows={`repeat(1,auto)`}
+          {...{ ref: drop }}
+        >
```

Comment:
I know you're just copying existing code, but spreading the ref seems to be a workaround from an older MUI version and isn't needed anymore. Would you be able to search the project for `{...{ ref: ` and replace them with a normal prop? At least for lines you've changed.